package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.62.25-RDM-5007-prerelease",
+  "version": "2.62.26-RDM-6716-prerelease",
```

Comment:
Is this relevant to the change?

Reply:
I have discussed with Ashley on the versioning... He advised me to use 2.62.26 for this change...